spanner/session.go

```diff
@@ -227,6 +228,11 @@ func (s *session) String() string {
 func (s *session) ping() error {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 	defer cancel()
+
+	// Start parent span that doesn't record.
+	_, span := octrace.StartSpan(ctx, "ping", octrace.WithSampler(octrace.NeverSample()))
```

Comment:
Should we rename `ping` to a convention name, e.g., `"cloud.google.com/go/spanner.ping"`? Does this matter?

Reply:
That's a good question. This is usually used for API methods so I'm not sure what the correct form is here given that this is a name that we're making up. @rghetia what do you think?

Reply:
"cloud.google.com/go/spanner.ping" is better than just simple 'ping' - it helps to distinguish in case if another library also generates span with the name 'ping'.

Reply:
Great, thanks. I've made the change.